src/hooks/directoryHooks/contactUsHooks/useGetContactUsHook.ts

```diff
@@ -14,9 +15,9 @@ export const useGetContactUsHook = (
     [GET_CONTACT_US_KEY, siteName],
     () => ContactUsService.getContactUs(siteName),
     {
-      onError: () => {
+      onError: (err: AxiosError) => {
         errorToast({
-          description: `Your Contact Us page details could not be retrieved. ${DEFAULT_RETRY_MSG}`,
+          description: `Your Contact Us page details could not be retrieved. ${DEFAULT_RETRY_MSG}. Error: ${err.message}`,
```

Comment:
sanity check here - why is this `err.message` but in the other, `err.response?.data.error.message` even when the type is identical?

Reply:
Oops i changed the file but didn't save it - fixed in https://github.com/isomerpages/isomercms-frontend/pull/1272/commits/17904cef5b32809997235e5929b468af114b1ce4

Reply:
actually i realised there's a `getAxiosErrorMessage` util which does this - we can do this in a subsequent PR or this one.